core/primitives/src/sharding.rs

```diff
@@ -562,8 +556,9 @@ impl ShardChunkHeader {
         &self,
         version: ProtocolVersion,
     ) -> Result<(), BadHeaderForProtocolVersionError> {
+        #[allow(deprecated)]
         const CONGESTION_CONTROL_VERSION: ProtocolVersion =
-            ProtocolFeature::CongestionControl.protocol_version();
+            ProtocolFeature::_DeprecatedCongestionControl.protocol_version();
```

Comment:
Remove `CONGESTION_CONTROL_VERSION`. Change code to assume `version >= CONGESTION_CONTROL_VERSION` is true